### PR TITLE
Correcting translations

### PR DESCRIPTION
**Pull request by MohamedRayenJmili**
This pull request includes the following changes:

Adjusted the displays to accommodate both translations, specifically addressing the length variations between English and French.
Incorporated the requested changes from previous tickets.
These changes aim to improve the user experience by ensuring that button displays are optimized for both English and French translations, preventing text overlap and maintaining a clean interface.

Thank you for your review.

### CODE REVIEW

File: src/assets/i18n/fr.json [hunk 1]
```diff
@@ -1084,8 +1084,8 @@
   "Facultative": "Facultative",
   "You-are-Requesting": "Vous demandez",
   "Send-again": "Réessayer autre fois",
-  "You-re-about-to-Send": "You’re about to Send",
-  "You-re-about-to-buy": "You’re about to Buy",
+  "You-re-about-to-Send": "Vous allez envoyer",
```
Comment (skanderKha) [inline src/assets/i18n/fr.json]: " Vous êtes sur le point d'envoyer " is better

File: src/assets/i18n/fr.json [hunk 2]
```diff
@@ -1268,7 +1268,21 @@
   "activate-oracle-tiktok-text2": " pour commencer à gagner des SaTT.",
   "tokensMigratedSuccessfully":"Migration des Tokens réussie",
   "checkYourWallet":"Vérifiez votre portefeuille: ",
-  "SATTBEP20": "SATT"
+  "SATTBEP20": "SATT",
+  "Accounts":"Comptes",
+  "activate-oracle-linkedin-text1": "Devenez un créateur de contenu Linkedin certifié pour commencer à gagner du SaTT.",
+  "setYourNewTransactionPasswordText1":"Veuillez saisir le nouveau mot de passe de transaction. Vous devrez effectuer toutes les transactions avec votre nouveau portefeuille. Notez-le soigneusement et conservez-le en lieu sûr !",
+  "setYourNewTransactionPasswordText2":"Ce nouveau mot de passe n'est valable que pour votre nouveau portefeuille. L'ancien mot de passe reste valable pour l'ancien portefeuille.",
+  "goToNewWallet":"Aller à Nouveau portefeuille",
```
Comment (skanderKha) [inline src/assets/i18n/fr.json]: "Allez vers le nouveau portefeuille" is better

File: src/assets/i18n/fr.json [hunk 3]
```diff
@@ -1268,7 +1268,21 @@
...
+  "searchToken":"Cherche de Tokens",
```
Comment (skanderKha) [inline src/assets/i18n/fr.json]: " rechercher un token " or " recherche de tokens "  the term " recherche " is more appropriate than cherche

File: src/app/wallet/wallet.component.html [hunk 4]
```diff
@@ -270,15 +270,15 @@ <h3 class="textpwd">{{ 'Create your Tron wallet' | translate }}</h3>
     <div class="d-flex justify-content-center welcome-paragraph">
       <p class="paragragh">
         {{
-          'Now you have the possibility to create your Tron ​​wallet. You Need to just enter your transactional password to create it '
+          ' Now you have the possibility to create your Tron ​​wallet. You Need to just enter your transactional password to create it'
```
Comment (skanderKha) [inline src/app/wallet/wallet.component.html]: can you please put this sentence with its translation in the json file inside a viarable eg : createTronWallet: <sentence here>

File: src/app/wallet/wallet.component.css [hunk 5]
```diff
@@ -2432,50 +2432,34 @@ p {
...
-
   font-style: normal;
   font-weight: 600 !important;
   font-size: 80%;
-
   align-items: center;
   text-align: center;
-  letter-spacing: 0.005em;
   border: none;
+  width: 50%;
```
Comment (skanderKha) [inline src/app/wallet/wallet.component.css:2449]: duplication of width , can you check it please 2443 and 2450

**Review by skanderKha — CHANGES_REQUESTED** [review]
There is some little details that should be modified, if you have any question feel free to reach me

**Comment by skanderKha** [comment]
@MohamedRayenJmili  i made some comments on your change can you please check them and update the code thanks

**Review by skanderKha — APPROVED** [review]

**Review by ksibisamir — APPROVED** [review]

**Pull request merged**